apps/store/src/utils/PageLink.ts

```diff
@@ -349,3 +353,19 @@ const REVIEWS_URL: Partial<Record<RoutingLocale, URL>> = {
   se: new URL('/se/hedvig/omdomen', ORIGIN_URL),
   'se-en': new URL('/se-en/hedvig/reviews', ORIGIN_URL),
 }
+
+export const removeTrailingSlash = (url: string) => {
+  return url.endsWith('/') ? url.slice(0, -1) : url
+}
+
+export const removeSEHomepageLangSegment = (urlString: string): string => {
+  const url = new URL(urlString)
+
+  const isSwedishHomepage = removeTrailingSlash(url.pathname) === '/se'
```

Comment:
Suggestion: Would the following work as well?
I found `removeTrailingSlash(url.pathname) === '/se'` line a little bit confusing because in theory `removeTrailingSlash` expects a full URL. But maybe it's just me :)

```suggestion
  const url = new URL(removeTrailingSlash(urlString))

  const isSwedishHomepage = url.pathname === '/se'
```